src/secp256k1/.cirrus.yml

```
@@ -125,20 +122,22 @@ task:
     HOMEBREW_NO_INSTALL_CLEANUP: 1
     # Cirrus gives us a fixed number of 4 virtual CPUs. Not that we even have that many jobs at the moment...
     MAKEFLAGS: -j5
-  matrix:
-    << : *ENV_MATRIX
   env:
```

YAMLlint (1.37.1)

[error] duplication of key "env" in mapping

(key-duplicates)